core/db/migration/migrate-db.sh

```diff
@@ -16,6 +16,7 @@ echo "Implementing Migrations"
 
 psql -c "select addcol('public','users', 'last_login', 'timestamp', 'current_timestamp');"
 psql -c "select addcol('public','media', 'fine_amount', 'real', '0.50');"
+psql -c "select addcol('public','inventory', 'icn', 'varchar(16)', '' );"
```

Comment:
You don't need to provide the last variable, if you wanted, it could be `psql -c "select addcol('public','inventory', 'icn', 'varchar(16)' );"`